content/support-matrix/delfin/dell-emc/vmax/_index.md

```diff
@@ -0,0 +1,354 @@
+      <tr>
+         <td>Model</td>
+         <td>Device Model</td>
+      </tr>
+      <tr>
+         <td>Firmware version</td>
+         <td>Firmware version</td>
+      </tr>
+      <tr>
+         <td>Status</td>
+         <td>It shows availability of device ex: offline, normal etc.</td>
+      </tr>
+      <tr>
+         <td>Serial Number</td>
+         <td>It is VMAX array id</td>
+      </tr>
+      <tr>
+         <td>Location</td>
+         <td>Device Location, is name-value pair having component type and component name</td>
+      </tr>
+      <tr>
+         <td>Total Capacity</td>
+         <td>System capacity</td>
+      </tr>
+      <tr>
+         <td>Used Capacity</td>
+         <td>Total used capacity</td>
+      </tr>
+      <tr>
+         <td>Free Capacity</td>
+         <td>Total free capacity</td>
+      </tr>
+      <tr>
+         <td>Raw Capacity</td>
+         <td>Raw capacity(sum of all disks capacity)</td>
+      </tr>
+      <tr>
+         <td>Subscribed Capacity</td>
+         <td>Total amount of capacity subscribed(bytes)</td>
+      </tr>
+   </tbody>
+</table>
+ 
+### Alert data matrix
+ 
+<table>
+   <thead>
+      <tr>
+         <th>Attribute</th>
+         <th>Additional information</th>
+      </tr>
+   </thead>
+   <tbody>
+      <tr>
+         <td>Alert id</td>
+         <td>Identification of alarm.</td>
+      </tr>
+      <tr>
+         <td>Alert name</td>
+         <td>Name of the alarm</td>
+      </tr>
+      <tr>
+         <td>Severity</td>
+         <td>Severity of the alarm</td>
+      </tr>
+      <tr>
+         <td>Category</td>
+         <td>Category of the reported alarm notification. Ex: Fault, Event or Recovery</td>
+      </tr>
+      <tr>
+         <td>Type</td>
+         <td>Basic classification of the alarm. Ex:CommunicationsAlarm, EquipmentAlarm</td>
+      </tr>
+      <tr>
+         <td>Sequence number</td>
+         <td>Unique identification for the alert instance</td>
+      </tr>
+      <tr>
+         <td>Occur time</td>
+         <td>Time at which alert is generated from device. Specified in epoch format</td>
+      </tr>
+      <tr>
+         <td>Description</td>
+         <td>Alarm Description</td>
+      </tr>
+      <tr>
+         <td>Recovery advice</td>
+         <td>This field does not contain any info. However description field may contain some recovery advice in some cases.</td>
+      </tr>
+      <tr>
+         <td>Resource type</td>
+         <td>Resource type for alert which can be storage subsystem or some other subsystems</td>
+      </tr>
+      <tr>
+         <td>Location</td>
+         <td>Detailed info about tracing the alerting device such as array id, component name, component type</td>
+      </tr>
+   </tbody>
+</table>
+ 
+### Performance metric data
+ 
+<table>
+   <thead>
+      <tr>
+         <th>Resource</th>
+         <th>Metric</th>
+         <th>Additional information</th>
+      </tr>
+   </thead>
+   <tbody>
+      <tr>
+         <td rowspan=3></td>
```

Comment:
Table is empty. Please remove if not available or applicable

Reply:
same as above